qa/tasks/cephfs/test_admin.py

```diff
@@ -86,6 +87,110 @@ def test_fs_status(self):
         self.assertEqual(mdsmap[0]["state"], "active")
 
 
+class TestAaaFSDown(TestAdminCommands):
+
+    MDSS_REQUIRED = 4
+
+    def get_num_of_standby_MDSs(self):
+        output = json.loads(self.get_ceph_cmd_stdout(
+            'mds stat --format json-pretty'))
+        num_of_standby_MDSs = len(output['fsmap']['standbys'])
+        return num_of_standby_MDSs
+
+    def get_num_of_MDSs(self):
+        output = json.loads(self.get_ceph_cmd_stdout(
+            'fs status --format json-pretty'))
+        num_of_MDSs = len(output['mdsmap'])
+        return num_of_MDSs
+
+    def wait_till_all_MDSs_are_down(self):
+        sleep(5)
+        num_of_MDSs = self.get_num_of_MDSs()
+        with safe_while(tries=120, sleep=5) as condition:
+            while condition():
+                cur_num_of_standby_MDSs = self.get_num_of_standby_MDSs()
+                if cur_num_of_standby_MDSs == num_of_MDSs:
+                    break
+
+    def test_fs_down_on_quincy(self):
```

Comment:
are these test cases available only in the `quincy` branch ?

Reply:
I didn't intend to merge them originally, they were only meant for BZ we discussed.  

But I guess it's a good idea to merge them, since they've been written and since they'll be useful.